### PR TITLE
RET-3055

### PR DESCRIPTION
**Pull request by ikirsanov**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RET-3055

### Change description ###
Changed retention period to 78 weeks

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by Itay-Kirsh — DISMISSED** [review]

**Review by cindychmcts — APPROVED** [review]

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -78,4 +78,67 @@
     <notes>Temporary suppression for commons-fileupload-1.4.jar</notes>
     <cve>CVE-2023-24998</cve>
   </suppress>
+  <suppress>
```
Comment (fergusoshea) [inline config/owasp/suppressions.xml:81]: Do we still need these after Paul's update yesterday?

Comment (JackReevies) [inline config/owasp/suppressions.xml:81]: This looks like a lot of suppressions, are we sure theres no update to these deps?

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/ApplicationService.java [hunk 2]
```diff
@@ -203,7 +203,7 @@ private JSONObject getDocumentDownload(String authorization, CaseData caseData)
                     downloadDocument.getByteArray(),
                     false,
                     true,
-                    "52 weeks"
+                    "78 weeks"
```
Comment (paul-peasegood) [inline src/main/java/uk/gov/hmcts/reform/et/syaapi/service/ApplicationService.java:206]: Whats this change for? Why is 78 weeks hardcoded

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (marcjudge) [reply config/owasp/suppressions.xml:81]: 1.2 needs to be updated from master, I will do that

Comment (marcjudge) [reply config/owasp/suppressions.xml:81]: Done

File: build.gradle [hunk 3]
```diff
@@ -251,7 +251,7 @@ dependencies {
   implementation group: 'org.springframework.security', name: 'spring-security-crypto', version: '5.7.6'
 
 
-  implementation group: 'com.fasterxml.jackson.core', name:  'jackson-databind', version: '2.14.2'
```
Comment (marcjudge) [inline build.gradle:254]: Do we need jackson-databind twice?

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (marcjudge) [reply config/owasp/suppressions.xml:81]: Please try removing these suppressions after getting the latest changes from 1.2

**Review by marcjudge — COMMENTED** [review]
I need to update 1.2

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/ApplicationService.java (hunk 2 above)
Comment (ikirsanov) [reply src/main/java/uk/gov/hmcts/reform/et/syaapi/service/ApplicationService.java:206]: NB By default the PDF can remain active in the email for up to 78 weeks. After this time the PDF link should be expired

Comment (paul-peasegood) [reply src/main/java/uk/gov/hmcts/reform/et/syaapi/service/ApplicationService.java:206]: Could the PDF expiry time be put into a constant? or even config?

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (ikirsanov) [reply config/owasp/suppressions.xml]: Fixed that

Comment (ikirsanov) [reply config/owasp/suppressions.xml]: Removed suppression after merge it fixed

File: build.gradle (hunk 3 above)
Comment (marcjudge) [reply build.gradle:254]: I don't think we do. Please remove it

**Review by marcjudge — CHANGES_REQUESTED** [review]

**Review by JackReevies — DISMISSED** [review]

File: build.gradle (hunk 3 above)
Comment (ikirsanov) [reply build.gradle:254]: done

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/ApplicationService.java (hunk 2 above)
Comment (ikirsanov) [reply src/main/java/uk/gov/hmcts/reform/et/syaapi/service/ApplicationService.java]: Done

**Review by fergusoshea — APPROVED** [review]

**Pull request merged**